Analysis/src/AnaTuple.cpp

```diff
@@ -194,6 +194,8 @@ void AnaTupleWriter::AddEvent(EventInfoBase& event, const AnaTupleWriter::DataId
                                                    t1.GetMomentum(), t2.GetMomentum(), b1.GetMomentum(),
                                                    b2.GetMomentum(), event.GetMET().GetMomentum()).second);
         tuple().HT_total = static_cast<float>(b1.GetMomentum().pt() + b2.GetMomentum().Pt() + event->ht_other_jets);
+        tuple().dR_lj = static_cast<float>(four_bodies::Calculate_dR(t1.GetMomentum(), t2.GetMomentum(), b1.GetMomentum(), b2.GetMomentum()));
```

Comment:
`Calculate_dR` is not committed to AnalysisTools. Also change the name to something more self-explanatory